source/geppetto/4.2/index.md

```diff
@@ -0,0 +1,286 @@
+3. Right-click the project and click **Team** -> **Share Project**.
+4. In the wizard, select **Git** and click **Next**.
+5. Select the repository in the drop-down list, enter the name of the repo folder where you want to put the project, and click **Finish**.
+
+For more information about interacting with Git or SVN from Geppetto, refer to the EGit and Subversive user guides. On the Geppetto **Help** menu, click **Help Contents** to open the guides.
+
+## Editing Module Metadata
+
+All Puppet modules include a metadata file, which contains high-level information about the module (such as version, author, license, and dependencies). You can modify this metadata through Geppetto's metadata editor. This editor features syntax coloring, tooltips, autocompletion, dependency lookups, and formatting. These features are all configurable in Geppetto's Preferences menu.  
+
+Open any module's metadata.json in Geppetto to open the editor and modify the metadata. If you create a module using Geppetto's Project Wizard, you will have access to the same metadata editor. 
+
+Upon pulling a module from the Puppet Forge, Geppetto will open the metadata.json in the editor automatically. 
+
+### Fields in metadata.json:
+
+* `name`: The name of your module. If you are planning to upload your module to the Puppet Forge at any point, your module name should begin with your username, i.e., "username-module". See [our guidelines on module naming](module_names) for more information. 
+* `version`: Your module's current version number. It is important that you keep this up-to-date, as it affects module dependencies. We highly recommend using [semver](http://semver.org/) to guide your versioning.
+* `author`: The name of the module's author.
+* `license`: Any applicable license guiding the module's use.
+* `summary: A brief description of the module's functionality.
+* `source`: The repo where the module lives. If you used Geppetto to create your module, you can ignore this.
+* `project_page`: A link to your module's website. If you are publishing to the Puppet Forge, this link typically points to your module on the Forge.
+* `issues_url`: A link to your module's issue tracker.
+* `operatingsystem_support`: A list of operating systems with which your module is compatible. Please see [operating system compatibility in metadata.json](os_metadatajson) for more information.
+* `tags`: Key words that will help others find your module, such as "mysql", "database", or "monitoring". Tags cannot contain whitespace and are case-insensitive.
+* `dependencies`: If your module requires capabilities or functionality provided by another module in order to run properly, your module is dependent on that other module. 
+You should express such dependencies in this field. For example, the [puppetlabs-postgresql](https://forge.puppetlabs.com/puppetlabs/postgresql) has these dependencies:
+
+  ```
```

Comment:
Replace triple-backticks with triple-tildes (~), for compatibility with the markdown renderer our website uses.